### PR TITLE
introduce feature for hgraph

### PR DESCRIPTION
**Pull request by LHT129**
issue: #154

### CODE REVIEW

File: src/index_feature_list.cpp [hunk 1]
```diff
@@ -31,11 +33,12 @@ IndexFeatureList::IndexFeatureList()
...
-IndexFeatureList::CheckFeature(IndexFeature feature) {
-    if (static_cast<uint32_t>(feature) >= feature_count_) {
-        throw std::runtime_error("wrong feature");
+IndexFeatureList::CheckFeature(IndexFeature feature) const {
+    auto val = static_cast<uint32_t>(feature);
+    if (val >= feature_count_ or val == 0) {
+        throw std::invalid_argument(fmt::format("wrong feature value: {}", val));
     } else {
-        auto [x, y] = get_pos(static_cast<uint32_t>(feature));
+        auto [x, y] = get_pos(val);
```
Comment (jiaweizone) [inline src/index_feature_list.cpp:41]: where is define for `get_pos` ?

Comment (LHT129) [reply src/index_feature_list.cpp:41]: https://github.com/antgroup/vsag/blob/main/src/index_feature_list.cpp#L23

**Review by inabao — APPROVED** [review]
LGTM

**Review by jiaweizone — APPROVED** [review]
LGTM

**Review by wxyucs — APPROVED** [review]
lgtm

**Review by ShawnShawnYou — APPROVED** [review]

**Pull request merged**